.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: go
 
 go:
-  - 1.6.3
-  - 1.7
+  - 1.8.x
+  - 1.9.x
```

Comment:
travis was failing without this change

Reply:
Could you also add Go 1.10 as a target while you're here?